package.json

```diff
@@ -1,10 +1,44 @@
 {
   "name": "client-side-validations",
-  "private": true,
+  "version": "12.0.0beta7",
```

Comment:
I'm not an expert of js modules and libraries. 

Why we don't need private?

From: https://docs.npmjs.com/files/package.json#license
> Consider also setting "private": true to prevent accidental publication.

client side validations can't work without its ruby counterpart, so I think this should not be published.

Also, if we need version in this file, it should be automated through the rake task (or whatever)

Reply:
The benefit to publishing the JS on NPM is that users don't have to have this libraries code checked into their source control. Once in NPM user will just have this added to the package.json:
`"client-side-validations": "^11.1.2",`
and then import it in there application:
`import 'client-side-validations'`
You can still keep the option to copy the file over, but for users accustomed to webpack and yarn/npm it feels more natural to have it added to the node_modules. Also, I agree with the version being added by the rake task, but not sure how to do that. I have minimal experience with rake.

Reply:
A good example of this is https://www.npmjs.com/package/rails-ujs

Reply:
> users don't have to have this libraries code checked into their source control. 

Client Side Validations is not a javascript library. CSV is a ruby library with a javascript counterpart. You cannot use npm without the gem, so you will need both.

This could lead to duplication and different versions used at the same time.

I'm not interested in extracting the javascript code to its own node module